src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -80,7 +80,7 @@ def _create_session(
         s.auth = auth if auth else s.auth
         s.proxies = proxies if proxies else s.proxies
 
-        max_retries = self.builtin.convert_to_integer(max_retries)        
+        max_retries = int(max_retries)        
```

Comment:
[The “Convert To Integer” is a Robot BuiltIn keyword.](http://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Convert%20To%20Integer) They _“can”_ be called from other keywords **but** will print misleading log messages that are not related to the task of keyword calling it (in this case the Create Session keyword).

Another way to avoid the misleading log messages is to invoke the inner method _convert_to_integer that do not print to the logs, instead of its wrapper convert_to_integer.

Therefore, the line of code would be like:
`max_retries = self.builtin._convert_to_integer(max_retries)`

If you prefer this solution, I can do a Pull Request with that change instead.